src/community/ogcapi/ogcapi-tiles/src/main/resources/org/geoserver/ogcapi/tiles/tileMatrixSets.ftl

```diff
@@ -1,5 +1,7 @@
 <#global pagetitle="Tile matrix sets">
-<#global pagecrumbs="<li class='breadcrumb-item'><a href='"+serviceLink("")+"'>Home</a></li><li class='breadcrumb-item'><a href='"+serviceLink("collections")+"'>Collections</a></li><li class='breadcrumb-item active'>"+pagetitle+"</li>">
+<#global pagecrumbs="<li class='breadcrumb-item'><a href='"+serviceLink("")+"'>Home</a></li>
+<!-- li class='breadcrumb-item'><a href='"+serviceLink("collections")+"'>Collections</a></li-->
```

Comment:
Uh?

Reply:
@aaime 
I'm Removing the commented part which I forgot to definitely delete.

Explaining my change:

The previous version was putting TileMatrixSets link under "Collections".
So something like:
Home / Collections / TileMatrixSets

I suspect it was a refuse from copy and paste from the author which initially committed that new ftl based on the Collections one.
TileMatrixSets should not be "children" of Collections ->
Home / TileMatrixSets

Does it make sense?

Reply:
Yes it does, thank you!